docker/index.yaml

```diff
@@ -122,27 +118,34 @@ artifactory.algol60.net/csm-docker/stable:
       - v3.9.3
 
     # CoreDNS required by platform
-    # XXX See also docker.io/coredns/coredns:1.6.2 above
+    # XXX Note this is different from any docker.io/coredns/coredns images
     k8s.gcr.io/coredns:
       - v1.8.0
 
+    # Note this is the new layout for k8s 1.22 for coredns upstream the above
+    # will go away for k8s 1.23+
+    k8s.gcr.io/coredns/coredns:
+      - v1.8.0
+      - v1.8.4
+
     # Kube images required by platform
     k8s.gcr.io/kube-apiserver:
-      - v1.20.13
       - v1.21.12
+      - v1.22.13
     k8s.gcr.io/kube-controller-manager:
-      - v1.20.13
       - v1.21.12
+      - v1.22.13
     k8s.gcr.io/kube-proxy:
-      - v1.20.13
       - v1.21.12
+      - v1.22.13
     k8s.gcr.io/kube-scheduler:
-      - v1.20.13
       - v1.21.12
```

Comment:
are we missing the 1.22.13 one for scheduler?

Reply:
Line 143 I moved that quay.io thing out of the k8s block.